lasagne/layers/__init__.py

```diff
@@ -8,4 +8,5 @@
 from .shape import *
 from .merge import *
 from .normalization import *
+from .prelu import *
```

Comment:
I don't think this should become its own submodule of just a single layer. We could have a submodule for parameterized (or otherwise complex) nonlinearities, though. What would be a concise name?